recipes/libbsd/all/conanfile.py

```diff
@@ -32,51 +36,70 @@ def build_requirements(self):
     def _source_subfolder(self):
         return "source_subfolder"
 
+    def export_sources(self):
+        export_conandata_patches(self)
+
     def configure(self):
         if self.options.shared:
             del self.options.fPIC
-        del self.settings.compiler.libcxx
-        del self.settings.compiler.cppstd
+        try:
+            del self.settings.compiler.libcxx
+        except ConanException:
+            pass
+        try:
+            del self.settings.compiler.cppstd
+        except ConanException:
+            pass
+    
+    def generate(self):
+        if not cross_building(self):
+            env = VirtualRunEnv(self)
+            env.generate(scope="build")
+        tc = AutotoolsToolchain(self)
+        if is_apple_os(self):
+            tc.extra_cflags.append("-Wno-error=implicit-function-declaration")
+        tc.generate()
+        deps = AutotoolsDeps(self)
+        deps.generate()
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
```

Comment:
This is not a Cmake project

```suggestion
        basic_layout(self, src_folder="src")
```